pom.xml

```diff
@@ -8,5 +8,28 @@
     <artifactId>springHW</artifactId>
     <version>1.0-SNAPSHOT</version>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <version>5.1.6.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>3.0.5.RELEASE</version>
```

Comment:
Why not version 5.1.*?
![image](https://user-images.githubusercontent.com/20267148/56947069-b7584e80-6b34-11e9-8d7f-233d61c9124d.png)